content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+## GoogleBenchmark
+
+For performance benchmarks of small pieces of C++ code O3DE uses [GoogleBenchmark](https://github.com/google/benchmark/blob/main/docs/index.md), which is quite similar to GoogleTest. However, the definition of failure is a major difference between a normal test and a benchmark. In most tests a pass/fail status is directly evaluated to a boolean state, creating an objective report of success and failure. Benchmarks instead create a subjective performance metric. These metrics are most valuable when periodically recorded, to help detect trends over time. The only objective failure during a benchmark occurs when the code fails to run or crashes.
+
+The steps for configuring a GoogleBenchmark library are identical to the [steps above for GoogleTest libraries](#googletest), with a couple exceptions. The include statement is `#include <benchmark/benchmark.h>` and the cmake helper function is:
+
+```
+ly_add_googlebenchmark(
+    NAME MyModule.Benchmarks
+)
+```
+
+## PyTest
+
+Certain tests are easier to write in a scripting language like Python. The scope of these tests are typically at the [integration level](https://softwaretestingfundamentals.com/integration-testing/), verifying mid-to-high level functionality of workflows. This type of test helps verify system correctness similar to how an end-user will experience the software. Despite that positive aspect, these tests are typically slow and provide less specific failure information. Not all Python-based tests are integration tests: any Python-based tools will prefer unit tests written in Python. For all tests written in Python, O3DE uses [PyTest](https://docs.pytest.org/).
+
+While interpreted languages like Python can have low performance, these tests should not perform heavy computation in Python. Tests should instead focus on coordinating workflows: signalling events and then verifying a response. Leave heavy operations to the code targeted by a test, and perform simple checks in Python test-code.
+
+There are often two Python interpreter instances used during O3DE tests. This can complicate determining which interpreter executes your code, so pay close attention to how the test is defined:
+
+- Test exists outside of **O3DE Editor**
+  - Use in generic tests that launch applications and send external signals.
+  - Helps monitor for application crashes.
+  - Test file names should start with `test_`.
+  - Uses the same Python interpreter that launches when you run `o3de/python/python.sh` or `o3de\python\python.cmd`.
+
+- Test embedded inside O3DE Editor
+  - Use in `EditorPythonBindings` tests which target workflows inside O3DE Editor.
+  - Helps expose editor functionality to a test script.
+  - Test file names should ***not*** include affixes such as `test_` or `_tests`.
+  - PyTest is not used inside the editor, and most PyTest functionality is not available
+    - PyTest is still initiated by the external Python interpreter instance, to handle Editor crashes.
```

Comment:
My attempt:

```suggestion
O3DE can run multiple Python interpreter instances for testing. Currently, there are two interpreters: The _Editor interpreter_ and the _external interpreter_. It can be difficult to determine which environment your Python tests execute in. To determine which environment your test should run in, and how to execute it there, follow the remainder of this section.

#### Tests in the external interpreter

The external interpreter runs outside of any active O3DE application. This is the same interpreter that launches when you run `o3de/python/python.sh` or `o3de\python\python.cmd`., and is best used for tests which involve the following:

- Generic tests that launch applications and send external signals.
- Monitoring application crashes.

To integrate with the external interpreter, your tests must be in filenames which start with `test_`.

#### Tests in the Editor interpreter

The Editor interpreter runs inside of O3DE Editor, using an internal Python library. Although this environment isn't equivalent to launching `o3de/python/python.*`, it uses the same version of the Python interpreter. The Editor interpreter is best used for tests which involve the following:

- Targeting specific functionality within the Editor or another application
- Exposing Editor functionality to another test

To integrate with the Editor interpreter, create a test which uses `EditorPythonBindings`. These tests **must not** be in a file starting with `test_` or `tests_`.

{{< note >}}
PyTest is not used within the Editor, and as a consequence PyTest functionality is unavailable to tests which run in the Editor interpreter. If your test requires the usage of PytTest, create a test which runs in the external interpreter and then monitors activity appropriately as reported by a test which runs in the Editor interpreter.
{{< /note >}}
```

The language used here for some of the details of in-Editor interpreter testing are what I was able to divine from the content as written; if they're not precise, make sure to update.

Reply:
Good suggestions, updating for accuracy